hack/update-codegen.sh

```diff
@@ -5,9 +5,9 @@ set -o nounset
 set -o pipefail
 
 SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
-CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ${GOPATH}/src/k8s.io/code-generator)}
+CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../code-generator)}
```

Comment:
but there is no `code-generator` subdirectory in the operator top-level dir or ?

Reply:
I'll change it back. Just copied it from [this](https://github.com/kubernetes/sample-apiserver/blob/f3d6226a155191940485b3f7a45742e1ca39c0b8/hack/update-codegen.sh#L22) example.